### DISCUSSION TITLE
Postgis GEOGRAPHY(POINT) type

### DISCUSSION THREAD

**Discussion by sergiohdljr** [porsager/postgres#780] [Q&A]
I have a table with a column of the type GEOGRAPHY(POINT)

this is how is made a insert in the pgadmin work just fine
sql ` INSERT INTO farmacia (nome, telefone, cidade, bairro, rua, numero, coords)
              VALUES ('Farmacia Espinheio', '40023633', 'Recife', 'Espinheiro',
              'Rua do Espinheiro', 222 , ST_GeographyFromText('POINT(-34.89536080032862 -8.04613923401633)'));`

but for some reason when i try to insert with postgres.js library, like that :
![Captura de tela de 2024-01-17 15-05-21](https://github.com/porsager/postgres/assets/102623806/fc5f3864-2a96-4e28-9a56-761b15aa595c)

i get the follow error : 
![Captura de tela de 2024-01-17 15-06-17](https://github.com/porsager/postgres/assets/102623806/66ce69c2-8462-487e-a5fd-7e8857078c1e)

I am very open to suggestions.

**Comment by thejasonxie** [comment]
I know this is late but refer to https://github.com/porsager/postgres/issues/503#issuecomment-1275605055
